SmartHomeManager/SmartHomeManager.API/Controllers/AccountAPI/AccountsController.cs

```diff
@@ -50,68 +46,63 @@ public async Task<ActionResult<IEnumerable<Account>>> GetAccounts()
 
             if (accounts == null)
```

Comment:
this expression is never true because if there are no accounts, it returns you an empty IEnumerable instead of null. You might wanna check for capacity instead. Might wanna convert to list first though.